src/lib/Microsoft.Health.Events/Telemetry/Exceptions/InvalidEventHubException.cs

```diff
@@ -13,15 +13,30 @@ public sealed class InvalidEventHubException : IomtTelemetryFormattableException
     {
         private static readonly string _errorType = ErrorType.EventHubError;
 
+        public InvalidEventHubException()
+        {
+        }
+
+        public InvalidEventHubException(string message)
+            : base(message)
+        {
+        }
+
+        public InvalidEventHubException(string message, Exception innerException)
+            : base(message, innerException)
+        {
+        }
+
         public InvalidEventHubException(
             string message,
             Exception innerException,
-            string errorName)
+            string errorName,
+            string operation)
             : base(
                   message,
                   innerException,
                   name: $"{_errorType}{errorName}",
-                  operation: ConnectorOperation.Setup)
```

Comment:
I actually preferred these exceptions being categorized as Setup as opposed to Normalization/FhirTransformation to differentiate from actual operation issues post setup. Just curious on why we want to change this?
Adding @wi-y to add any thoughts as some of the earlier related metrics were also intentionally set with Setup.